.github/workflows/health_check.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Notify slack
         uses: kpritam/slack-job-status-action@v1
         with:
-          if: failure()
+          if: ${{ failure() }}
```

Comment:
The Slack notifications were not happening for the health check. Testing to see if this works better.

Reply:
hmm shouldn't need to do it this way but worth a shot